Tests fail with pytest 4.6.3

They worked with pytest 4.2.1, but since 4.6.3 I get the following test failures:

```

=================================== FAILURES ===================================
_________________________ test_detailed_introspection __________________________

testdir = <Testdir local('/build/pytest-of-nixbld/pytest-0/test_detailed_introspection0')>

    @pytest.mark.usefixtures("needs_assert_rewrite")
    def test_detailed_introspection(testdir):
        """Check that the "mock_use_standalone" is being used.
        """
        testdir.makepyfile(
            """
            def test(mocker):
                m = mocker.Mock()
                m('fo')
                m.assert_called_once_with('', bar=4)
        """
        )
        result = testdir.runpytest("-s")
        if NEW_FORMATTING:
            expected_lines = [
                "*AssertionError: expected call not found.",
                "*Expected: mock('', bar=4)",
                "*Actual: mock('fo')",
            ]
        else:
            expected_lines = [
                "*AssertionError: Expected call: mock('', bar=4)*",
                "*Actual call: mock('fo')*",
            ]
        expected_lines += [
            "*pytest introspection follows:*",
            "*Args:",
            "*assert ('fo',) == ('',)",
            "*At index 0 diff: 'fo' != ''*",
            "*Use -v to get the full diff*",
            "*Kwargs:*",
            "*assert {} == {'bar': 4}*",
            "*Right contains more items:*",
            "*{'bar': 4}*",
            "*Use -v to get the full diff*",
        ]
>       result.stdout.fnmatch_lines(expected_lines)
E       Failed: nomatch: "*AssertionError: Expected call: mock('', bar=4)*"
E           and: '============================= test session starts =============================='
E           and: 'platform linux -- Python 3.7.3, pytest-4.6.3, py-1.7.0, pluggy-0.12.0'
E           and: 'rootdir: /build/pytest-of-nixbld/pytest-0/test_detailed_introspection0'
E           and: 'plugins: mock-0.0.0'
E           and: 'collected 1 item'
E           and: ''
E           and: 'test_detailed_introspection.py F'
E           and: ''
E           and: '=================================== FAILURES ==================================='
E           and: '_____________________________________ test _____________________________________'
E           and: ''
E           and: 'mocker = <pytest_mock.MockFixture object at 0x7ffff62d89e8>'
E           and: ''
E           and: '    def test(mocker):'
E           and: '        m = mocker.Mock()'
E           and: "        m('fo')"
E           and: ">       m.assert_called_once_with('', bar=4)"
E       fnmatch: "*AssertionError: Expected call: mock('', bar=4)*"
E          with: "E       AssertionError: Expected call: mock('', bar=4)"
E       fnmatch: "*Actual call: mock('fo')*"
E          with: "E       Actual call: mock('fo')"
E       nomatch: '*pytest introspection follows:*'
E           and: 'E       '
E       fnmatch: '*pytest introspection follows:*'
E          with: 'E       pytest introspection follows:'
E       nomatch: '*Args:'
E           and: 'E       '
E       fnmatch: '*Args:'
E          with: 'E       Args:'
E       fnmatch: "*assert ('fo',) == ('',)"
E          with: "E       assert ('fo',) == ('',)"
E       fnmatch: "*At index 0 diff: 'fo' != ''*"
E          with: "E         At index 0 diff: 'fo' != ''"
E       fnmatch: '*Use -v to get the full diff*'
E          with: 'E         Use -v to get the full diff'
E       fnmatch: '*Kwargs:*'
E          with: 'E       Kwargs:'
E       fnmatch: "*assert {} == {'bar': 4}*"
E          with: "E       assert {} == {'bar': 4}"
E       nomatch: '*Right contains more items:*'
E           and: 'E         Right contains 1 more item:'
E           and: "E         {'bar': 4}"
E           and: 'E         Use -v to get the full diff'
E           and: ''
E           and: 'test_detailed_introspection.py:4: AssertionError'
E           and: '=========================== 1 failed in 0.03 seconds ==========================='
E           and: ''
E       remains unmatched: '*Right contains more items:*'

/build/source/test_pytest_mock.py:646: Failed
----------------------------- Captured stdout call -----------------------------
============================= test session starts ==============================
platform linux -- Python 3.7.3, pytest-4.6.3, py-1.7.0, pluggy-0.12.0
rootdir: /build/pytest-of-nixbld/pytest-0/test_detailed_introspection0
plugins: mock-0.0.0
collected 1 item

test_detailed_introspection.py F

=================================== FAILURES ===================================
_____________________________________ test _____________________________________

mocker = <pytest_mock.MockFixture object at 0x7ffff62d89e8>

    def test(mocker):
        m = mocker.Mock()
        m('fo')
>       m.assert_called_once_with('', bar=4)
E       AssertionError: Expected call: mock('', bar=4)
E       Actual call: mock('fo')
E       
E       pytest introspection follows:
E           and: 'test_detailed_introspection.py:4: AssertionError'
E           and: '=========================== 1 failed in 0.03 seconds ======>
E           and: ''
E       remains unmatched: '*Right contains more items:*'

E           and: 'test_detailed_introspection.py:4: AssertionError'
E           and: '=========================== 1 failed in 0.03 seconds ==========================='
E           and: ''
E       remains unmatched: '*Right contains more items:*'

/build/source/test_pytest_mock.py:646: Failed
----------------------------- Captured stdout call -----------------------------
============================= test session starts ==============================
platform linux -- Python 3.7.3, pytest-4.6.3, py-1.7.0, pluggy-0.12.0
rootdir: /build/pytest-of-nixbld/pytest-0/test_detailed_introspection0
plugins: mock-0.0.0
collected 1 item

test_detailed_introspection.py F

=================================== FAILURES ===================================
_____________________________________ test _____________________________________

mocker = <pytest_mock.MockFixture object at 0x7ffff62d89e8>

    def test(mocker):
        m = mocker.Mock()
        m('fo')
>       m.assert_called_once_with('', bar=4)
E       AssertionError: Expected call: mock('', bar=4)
E       Actual call: mock('fo')
E       
E       pytest introspection follows:
E       
E       Args:
E       assert ('fo',) == ('',)
E         At index 0 diff: 'fo' != ''
E         Use -v to get the full diff
E       Kwargs:
E       assert {} == {'bar': 4}
E         Right contains 1 more item:
E         {'bar': 4}
E         Use -v to get the full diff

test_detailed_introspection.py:4: AssertionError
=========================== 1 failed in 0.03 seconds ===========================
=========================== short test summary info ============================
FAILED test_pytest_mock.py::test_detailed_introspection - Failed: nomatch: "*...
===================== 1 failed, 49 passed in 2.74 seconds ======================
```

--- a/test_pytest_mock.py
+++ b/test_pytest_mock.py
@@ -639,7 +639,7 @@ def test_detailed_introspection(testdir):
         "*Use -v to get the full diff*",
         "*Kwargs:*",
         "*assert {} == {'bar': 4}*",
-        "*Right contains more items:*",
+        "*Right contains* more item*",
         "*{'bar': 4}*",
         "*Use -v to get the full diff*",
     ]

--- a/tox.ini
+++ b/tox.ini
@@ -10,7 +10,7 @@ commands =
 
 [testenv:norewrite]
 commands =
-    pytest test_pytest_mock.py --assert=plain -ra
+    pytest test_pytest_mock.py --assert=plain
 
 [testenv:linting]
 skipsdist = True
@@ -20,7 +20,7 @@ basepython = python3.6
 commands = pre-commit run --all-files --show-diff-on-failure
 
 [pytest]
-addopts = -ra
+addopts = -r a
 
 [flake8]
 max-line-length = 88

